sqls.php

```diff
@@ -1,45 +1,87 @@
 <?php
 function countAllProjects() {
-    $query = 'SELECT count(*) FROM projects';
-    $result = pg_query($query) or die('Query failed: ' . pg_last_error());
-    $data = pg_fetch_row($result)[0];
-    pg_free_result($result);
-    return $data;
+  $query = 'SELECT count(*) FROM projects';
+  $result = pg_query($query) or die('Query failed: ' . pg_last_error());
+  $data = pg_fetch_row($result)[0];
+  pg_free_result($result);
+  return $data;
 }
 
 function countAllOnGoingProjects() {
-    return countAllProjects();
+  return countAllProjects();
 }
 
 function getProjectNames() {
-    $query = 'SELECT p.name, p.contents, u.username FROM projects p, users u WHERE p.owner = u.username';
+    $query = 'SELECT p.project_id, p.title, p.description, u.username, p.blurb FROM projects p, users u WHERE p.creator = u.username';
     $result = pg_query($query) or die('Query failed: ' . pg_last_error());
 
     while($row = pg_fetch_row($result)) {
         echo '<div class="col-md-4">';
-        echo '<h2>' . $row[0] . '</h2>';
-        echo '<div class="small">' . $row[1] . '</div>';
-        echo '<div><a>' . $row[2] . '</a></div>';
+        echo '<img src="' .$row[4] . '" style="max-width:100%;" />';
+        echo '<div class="panel panel-default"> <div class="panel-body" style="padding: 0 20px">';
+        echo '<h3><a href="projectdetails.php?project=' . $row[0] . '">' . $row[1] . '</a></h3>';
+        echo '<div>by <a>' . $row[3] . '</a></div>';
+        echo '<br /><br />';
+        echo '<div class="small ellipsis">' . $row[2] . '</div>';
+        echo '<br /><br />';
+        echo '<div class="progress"><div class="progress-bar" role="progressbar" aria-valuemin="0" aria-valuemax="100" style="width:' . 70 . '%"></div></div>';
+        echo '<br />';
+        echo '</div></div>';
         echo '</div>';
     }
     pg_free_result($result);
 }
 
 function isValidUser($username) {
-    $query = "SELECT u.username FROM users u WHERE u.username = '$username'";
-    $result = pg_query($query) or die('Query failed: ' . pg_last_error());
-    $bool = pg_num_rows($result) == 1;
-    pg_free_result($result);
-    echo $bool;
-    return $bool;
+  $query = "SELECT u.username FROM users u WHERE u.username = '$username'";
+  $result = pg_query($query) or die('Query failed: ' . pg_last_error());
+  $bool = pg_num_rows($result) == 1;
+  pg_free_result($result);
+  echo $bool;
```

Comment:
I think this shouldn't have the echo.

Because we might just want to know the results but not display it. Echo will bring it out to the html.